.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### What this PR does
+
+Fixes #<insert issue number here if relevant>
+
+This is a description of what I've done in this PR, especially explaining choices made to make the reviewer's job easier. If I don't replace this paragraph with an actual description, my PR will probably be ignored.
+
+### Checklist
+
+-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
+-   [ ] I've updated CHANGES.md with what I changed.
+-   [ ] I've linked this PR to an issue in ZenHub (core dev team only)
```

Comment:
Is this necessary if we're already linking to a ticket number above ?

And right now I can't see zenhub :joy:

Reply:
Link to ticket is optional but we don't actually use that part of zenhub atm, so I've removed it